### PR TITLE
Enable Chromatic again

### PR DESCRIPTION
**Pull request by scottrippey**
This reverts commit 8ac47935bf43009e9a3169108f22fa51c54b4ecb.

# TODO:
- [x] Ensure the "Victory Turbo" stories aren't being captured yet

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@ on:
     branches:
       - main
     paths:
-      - "stories/**"
+      - 'stories/**'
+      - 'packages/**'
```
Comment (ryan-roemer) [inline .github/workflows/chromatic.yml:11]: Should this be:

```suggestion
      - 'packages/*/src/**/*.stories.*'
```

for specificity?

Comment (ryan-roemer) [reply .github/workflows/chromatic.yml:11]: 🤔 or should we just enable it as is for really any `src` changes in case that impacts Storybook?

Comment (scottrippey) [reply .github/workflows/chromatic.yml:11]: This pattern is actually used for marking _when_ to trigger Chromatic, which we want to do any time we change `packages/**`.

We do have a glob pattern in `.storybook/main.js` which finds all stories to include. 
I'll update it to EXCLUDE all the `v37` stories that shouldn't be included yet.

**Comment by scottrippey** [comment]
Ironically, since this PR doesn't change any `packages/**`, the Chromatic tests didn't run :)
But the more I think about it, we should probably be running Chromatic even if root-level files are changed, since we don't want to introduce breaking builds.

**Review by ryan-roemer — APPROVED** [review]
🎉

**Pull request merged**